playbooks/switches_config.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Switches configuration requirements
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    # end_play will end this current playbook and go the the next
+    # imported play.
+    - name: Early playbook stop
+      when:
+        - not cifmw_configure_switches | default(true)
+      ansible.builtin.meta: end_play
+
+    - name: Install required packages
+      delegate_to: localhost
+      ansible.builtin.pip:
+        name:
+          - paramiko
+          - ncclient
+        extra_args: --user
+
+- name: Switches configuration
```

Comment:
ah, you need to end_play *all* of the plays listed in that file. Check ceph.yml, we're facing the same.

Also, using `pre_tasks` ensures a really early end.